Docs/alert.html

```diff
@@ -0,0 +1,113 @@
+                <li class="sidebar-items">
+                    <a href="">Input</a>
+                </li>
+            </ul>
+        </aside>
+        <main class="main-section flex-column">
+            <section>
+                <h1 class="headings main-heading font-weight-4">Alerts</h1>
+                <p class="heading-about font-weight-3"> 
+                    Alerts are used to attract user's attention for important information without interrupting the user's flow.
+                </p>
+            </section>
+            <section class="component-section flex-column">
+                <div>
+                    <h2 class="heading sub-heading font-weight-4"> Example of Alert Bar</h2>
+                    <p class="component-info">
+                        Alerts are available in 5 types- success alert, error alert, warning alert, primary alert, secondary alert.
+                        To use this alert just use class name alert-box and also add class according to alert type- 
+                        <span class="component-type">alert-primary</span>, <span class="component-type">alert-secondary</span>, <span class="component-type">alert-danger</span>, <span class="component-type">alert-success</span>, <span class="component-type">alert-warning</span>. (e.g.<span class="component-type">class="alert-box alert-primary"</span>).
+                        <br>
+                        You can copy html part from below code snippet.
+                    </p>
+                </div>
+
+                <div class="component-display-box font-weight-5">
+                    <div class="alert-box alert-primary">
+                        A primary alert example!!!
+                    </div>
+                    <div class="alert-box alert-secondary">
+                        A secondary alert example!!!
+                    </div>
+                    <div class="alert-box alert-danger">
+                        <span class="icons">
+                            <i class="fas fa-skull-crossbones"></i>
+                        </span>
+                        A danger alert example!!!
+                    </div>
+                    <div class="alert-box alert-success">
+                        <span class="icons">
+                            <i class="fas fa-check-circle"></i>
+                        </span>
+                        A success alert example!!!
+                    </div>
+                    <div class="alert-box alert-warning">
+                        <span class="icons">
+                            <i class="fas fa-exclamation-circle"></i>
+                        </span>
+                        A warning alert example!!!
+                    </div>
+                </div>
+                
+                <iframe src="https://carbon.now.sh/embed/Mho07EGvDAAOrQ3FXOJs" style="width: 100%; height: 400px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin"></iframe>
```

Comment:
You can avoid inline styling here and maybe instead include the CSS in a css file

Reply:
This is carbon iframe which I have imported form there

Reply:
yes I understand its carbon iframe you can create a separate class including these styles and include the class in the iframe tag @sri-rishi

Reply:
Did both of them you told, you can check again

Reply:
checked. Looks good 👍